dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -485,41 +486,42 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
     end
   end
 
-  test 'cannot assign an invalid course id' do
+  # Check this
+  test 'cannot assign an invalid course version id' do
     sign_in @teacher
     post :create, params: {
       login_type: Section::LOGIN_TYPE_EMAIL,
-      course_id: @beta_unit_group.id, # Not CSP or CSD
+      course_version_id: -1,
```

Comment:
This test's name doesn't seem to match this but `@beta_unit_group` looks like it's a valid unit group, just not an assignable one. Is there a reason this is changed?